client/app/lib/components/IconRadio.tsx

```diff
@@ -9,58 +9,40 @@ interface IconRadioProps {
   disabled?: boolean;
 }
 
-const styles: { [key: string]: CSSProperties } = {
-  icon: {
-    marginLeft: 1,
-    marginRight: 1,
-  },
-  radio: {
-    paddingLeft: 0,
-  },
-  container: {
-    padding: '0.5rem 0',
-    cursor: 'pointer',
-    display: 'flex',
-  },
-};
+const IconRadio = (props: IconRadioProps): JSX.Element => (
+  <label
+    className={`py-2 cursor-pointer flex ${
+      props.description ? 'items-start' : 'items-center'
+    }`}
+  >
+    <Radio value={props.value} className="pl-0" disabled={props.disabled} />
 
-const IconRadio = (props: IconRadioProps): JSX.Element => {
-  return (
-    <label
-      style={{
-        ...styles.container,
-        alignItems: props.description ? 'flex-start' : 'center',
-      }}
-    >
-      <Radio value={props.value} sx={styles.radio} disabled={props.disabled} />
+    {props.icon
+      ? createElement(props.icon, {
+          fontSize: 'large',
+          className: 'mx-4',
+          color: props.disabled ? 'disabled' : undefined,
+        })
+      : null}
 
-      {props.icon
-        ? createElement(props.icon, {
-            fontSize: 'large',
-            sx: styles.icon,
-            color: props.disabled ? 'disabled' : undefined,
-          })
-        : null}
+    <div>
```

Comment:
Replace div?

Reply:
To? The `div` is required here to arrange the `IconRadio`'s title and description vertically because the entire `label` element is a row flexbox.

Reply:
As discussed offline, we dont use Box component for simple div thanks!